packages/search-input/src/SearchResultGroup/SearchResultGroup.tsx

```diff
@@ -25,10 +25,12 @@ export const SearchResultGroup = ({
 
   return (
     <div>
+      {/* @ts-expect-error */}
```

Comment:
I assume this is TS because `li`'s props aren't compatible, but it would be good to note it here